### DISCUSSION TITLE
get rid of depracation warnings in trt model prep?

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#7900] [Ask A Question]
is it possible to to address the depracation warnings in the TRT model prep step?  

```
frigate           | /usr/local/src/tensorrt_demos/yolo/onnx_to_tensorrt.py:147: DeprecationWarning: Use network created with NetworkDefinitionCreationFlag::EXPLICIT_BATCH flag instead.
frigate           |   builder.max_batch_size = MAX_BATCH_SIZE
frigate           | /usr/local/src/tensorrt_demos/yolo/onnx_to_tensorrt.py:149: DeprecationWarning: Use set_memory_pool_limit instead.
frigate           |   config.max_workspace_size = 1 << 30
frigate           | /usr/local/src/tensorrt_demos/yolo/onnx_to_tensorrt.py:172: DeprecationWarning: Use build_serialized_network instead.
frigate           |   engine = builder.build_engine(network, config)
```

Maybe @NateMeyer can comment, this is on my Pascal based device.

**Comment by NateMeyer** [comment]
These come from the scripts in the repo here: https://github.com/NateMeyer/tensorrt_demos.  I wasn't the source of these, I just modified the one script in there.  Another thing to consider is if updating the scripts would cause an issue with the Jetson builds?  I would suggest making an issue in my tensorrt_demos repo, and a PR would be welcome.  I too have a Pascal card, but I couldn't test running these on the Jetson builds.

**Reply by FigJam23** [reply]
I get these warning exactly the same, won't generate models and sytem hangs. I can get the default yolo model to generate on first boot. But if I add a new yolo version to the environment I get those errors every time and no frigate start and models fail to be populated into the model dir

**Reply by Vacxe** [reply]
@FigJam23 have you fixed it? I facing the same issue.

**Reply by FigJam23** [reply]
> @FigJam23 have you fixed it? I facing the same issue.

Think I deleted all the models inside dir and rebooted, by memory it forces frigate to rebuild the models, took a bit of time to come up just be patient, can view the progress vial logs
